modules/@apostrophecms/image/ui/apos/components/AposMediaUploader.vue

```diff
@@ -128,17 +130,15 @@ export default {
         });
 
         // Send up placeholders
-        for (const file of files) {
-          this.createPlaceholder(file);
-        }
+        [ ...files ].forEach(() => this.createPlaceholder(emptyDoc));
 
-        const imageIds = [];
+        const images = [];
         // Actually upload the images and send them up once all done.
         for (const file of files) {
           try {
-            const img = await this.insertImage(file, emptyDoc);
+            const img = await this.insertImage(file, { ...emptyDoc });
```

Comment:
Do you need a full clone here?

Reply:
It mutates the object, so yes.

Reply:
ok, maybe use klona if you need a nested copy, if only for first level properties we good